### ISSUE TITLE
2015 commits

### ISSUE THREAD

**Issue by Pau1fitz** [pikesley/pokrovsky#31]
Hey,

I tried to run this but all the commits were dated 2015 - http://pokrovsky.herokuapp.com/pau1fitz/gity/2016

Does something need to be updated in the code?

**Comment by DenisBalan** [comment]
That's correct because it is working via creating past dated commits to fit in github's activity chart.
